packages/algob/src/lib/tx.ts

```diff
@@ -53,10 +53,10 @@ export function makeAssetCreateTxn (
     asaDef.total,
     asaDef.decimals,
     asaDef.defaultFrozen,
-    asaDef.manager,
-    asaDef.reserve,
-    asaDef.freeze,
-    asaDef.clawback,
+    asaDef.manager !== "" ? asaDef.manager : undefined,
+    asaDef.reserve !== "" ? asaDef.reserve : undefined,
+    asaDef.freeze !== "" ? asaDef.freeze : undefined,
+    asaDef.clawback !== "" ? asaDef.clawback : undefined,
```

Comment:
Can we do this during parsing of `asaDefs` (i.e when we read `" "` from YAML, we set it to undefined). Otherwise we would need to do repeat this each time for an asset related tx (eg. `AssetConfig`)

Reply:
yes

Reply:
thanks